core/queries/location/get/get_locations_query_handler.py

```diff
@@ -9,22 +7,17 @@
 from core.data_access.repositories.location_repository import LocationRepository
 from core.dtos.location_dto import LocationDto
 from core.queries.location.get.get_locations_query import GetLocationsQuery
-from core.serializers.location.get_locations_query_serializer import GetLocationsQuerySerializer
+
 
 class GetLocationsQueryHandler(RequestHandler[GetLocationsQuery, Result[List[LocationDto]]]):
     def __init__(self):
         self.location_repository = LocationRepository(Location)
         self.logger = logging.getLogger(__name__)
+        print("GetLocationsQueryHandler initialized")
```

Comment:
Why print? should we implement logger at some point instead?

Reply:
Forgot to remove